_source/_posts/2022-04-28-gdpr-and-app-security.md

```diff
@@ -0,0 +1,148 @@
+
+### Data mapping
+
+[Data mapping](https://en.wikipedia.org/wiki/Data_mapping), or connecting information from multiple data sets, requires a top-down analysis of all of an organization's databases and what information they hold. This can be a big task, but it ensures transparency and accountability. Data mapping is required for the next steps.
+
+Data mapping can be manual, semi-automated, or automated. You should determine which approach works best for you, depending on the size of your company, your budget, the quality of your data management team, and the amount of data you work with. Deciding how to [implement data mapping](https://blog.hubspot.com/marketing/data-mapping) will require some understanding of your databases, as well as knowing which duplicated columns can be optimized to reduce redundancy.
+
+### Data purge functionality
+
+Previously, even after users requested that their data be deleted, organizations could remove it from public access and keep it somewhere hidden; however, the GDPR mandates that "officially" deleted data must be deleted everywhere. Organizations must ensure that they monitor data to delete it properly from all resources as needed.
+
+Data purge functionality [implementation](https://success.outsystems.com/Documentation/Best_Practices/Architecture/Data_Purging) will likely require some modification to your database schema so that you can identify data to mark for deletion, per user requests, and data to purge due to staleness. A `LastUpdatedDate` and `IsDeleted` column in your data table can be useful in identifying either type of data.
+
+### Refactoring DB schemas
+
+Data purge functionality may involve refactoring [database (DB) schemas](https://en.wikipedia.org/wiki/Database_schema) to add columns noting when a resource was created or whether it was deleted. These columns can be used to automate data purges based on whether the data has been held longer than needed, for instance.
+
+Further DB schema analysis may also be required to ensure that more data isn't being collected than absolutely necessary.
+
+## Compliance is worth the work
+
+GDPR rules can be long and esoteric, but it's important for your organization to understand how to stay in compliance with the policy. Not only will you avoid fines for violations, but users will be more willing to trust your application. Ensuring better data privacy helps all stakeholders.
+
+If necessary, remember you can seek help from a GDPR officer to ensure compliance. Doing this work upfront can save your organization from more costly consequences later on.
+
+Check out these posts and whitepapers from Okta for more information about GDPR:
+
+- [Starting Your General Data Protection Regulation (GDPR) Journey with Okta](/whitepaper/starting-your-general-data-protection-regulation-journey-with-okta/)
```

Comment:
This link is 404.

Reply:
all links seems to be 404, i'll correct them